File: OCH_Tao/ABC/B/179.py
N = int(input())
L = []
for i in range(N):
  X,Y = map(int,input().split())
  L.append(X==Y)
cnt = 0
for i in L:
  if i:
    cnt+=1
  else:
    cnt=0
  if cnt==3:
    print("Yes")
    break
else:
  print("No")

File: OCH_Tao/ABC/B/179.js
const input = require("fs").readFileSync("/dev/stdin", "utf8").trim().split("\n");
input.shift();
let cnt = 0;
for (const i of input) {
  const [X, Y] = i.split(" ");
  if (X === Y) {
    cnt++;
  } else {
    cnt = 0;
  }
  if (cnt === 3) {
    console.log("Yes");
    return false;
  }
}
console.log("No");